pom.xml

```diff
@@ -89,6 +80,68 @@
         <module>web</module>
         <module>client-hc</module>
     </modules>
+    <dependencyManagement>
+        <dependencies>
+            <!-- provides compatible versions for jackson, guava, slf4j etc. -->
+            <dependency>
+                <groupId>io.dropwizard</groupId>
+                <artifactId>dropwizard-bom</artifactId>
+                <!-- make sure the dropwizard version is compatible to the below dropwizard-configurable-assets-bundle version -->
+                <version>1.3.12</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>io.dropwizard-bundles</groupId>
+                <artifactId>dropwizard-configurable-assets-bundle</artifactId>
+                <version>1.3.5</version>
+            </dependency>
+            <dependency>
+                <groupId>com.graphhopper.external</groupId>
+                <artifactId>jackson-datatype-jts</artifactId>
+                <version>0.12-2.5-1</version>
+            </dependency>
+            <dependency>
+                <groupId>com.carrotsearch</groupId>
+                <artifactId>hppc</artifactId>
+                <version>0.8.1</version>
+            </dependency>
+            <dependency>
+                <groupId>org.locationtech.jts</groupId>
+                <artifactId>jts-core</artifactId>
+                <version>1.15.1</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-compress</artifactId>
+                <version>1.18</version>
+            </dependency>
+            <dependency>
+                <groupId>log4j</groupId>
+                <artifactId>log4j</artifactId>
+                <version>1.2.17</version>
+            </dependency>
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-log4j12</artifactId>
+                <version>1.7.26</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>4.12</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
I added this here with the downside that all child projects have this dependency automatically now. Then again junit is used by all children and its only test scope too. With junit 5 this will be multiple dependencies, which otherwise we would have to repeat in every child pom.

Reply:
Interesting, I did the exact opposite a while ago. :-) I don't oppose at the moment, but let's remember that this is *only* for convenience: There ist absolutely zero dependency between the test frameworks each submodule uses (each one can use its own), so in a way this is an artificial restriction.

Reply:
Yes its code re-use via inheritance in a way, use with care... If you want we can of course go back to declaring the dependency in the submodules for the reason you mention.

Reply:
>  in a way this is an artificial restriction 

I don't want to be pedantic but a submodule doesn't not inherit dependencies from a parents dependencyManagement section and is still free to override the dependency versions in its own pom.

Reply:
Yes, but the junit dependency is an actual dependency (not dependencyManagement)

Reply:
> Yes, but the junit dependency is an actual dependency 

Sorry, my bad. 

> I think we all agree that using the dependencyManagement in the parent pom makes sense

Absolutely